docs/migrating/migrating_from_ez_publish_platform.md

```diff
@@ -20,7 +20,6 @@ You can then proceed with consecutive upgrades to further versions: v1.13 LTS an
     1. Additionally there are some other topics to be aware of for the code migration from eZ Publish to eZ Platform:
 
         - [Field Types reference](../api/field_type_reference.md) for overview of Field Types that do and don't exist in eZ Platform
-        - eZ Platform RichText Field Type capabilities, currently not covering [Custom Tags](https://jira.ez.no/browse/EZP-25357)
         - Symfony 2.8, this is also the case on later 5.4.x versions, but not the first ones including 2014.11
```

Comment:
we can update this too btw (when we talk migration to 2.5 it implies move to Symfony 3.4) to something like:
- Symfony 3.4LTS _(on v2.x)_